src/index.js

```diff
@@ -7,116 +7,91 @@ const {
+    formData: {
+      'account[mail]': username,
+      'account[pass]': password
+    },
     validate: (statusCode, $) => {
-      // The login in toscrape.com always works excepted when no password is set
-      if ($(`a[href='/logout']`).length === 1) {
+      if (
+        $(`a[href='https://www.glisshop.com/customers/logout/']`).text() ===
+        'Déconnexion'
+      ) {
         return true
       } else {
-        // cozy-konnector-libs has its own logging function which format these logs with colors in
-        // standalone and dev mode and as JSON in production mode
         log('error', $('.error').text())
         return false
       }
     }
   })
 }
 
-// The goal of this function is to parse a html page wrapped by a cheerio instance
-// and return an array of js objects which will be saved to the cozy by saveBills (https://github.com/konnectors/libs/blob/master/packages/cozy-konnector-libs/docs/api.md#savebills)
 function parseDocuments($) {
-  // you can find documentation about the scrape function here :
-  // https://github.com/konnectors/libs/blob/master/packages/cozy-konnector-libs/docs/api.md#scrape
   const docs = scrape(
     $,
     {
-      title: {
-        sel: 'h3 a',
-        attr: 'title'
+      commandNumber: {
+        sel: 'div[class=order-label] span',
+        parse: string => string.slice(9)
       },
       amount: {
-        sel: '.price_color',
-        parse: normalizePrice
+        sel: '.order-amount',
+        // Warning : Not a standard space
+        parse: amount => parseFloat(amount.split(' ')[0])
       },
-      fileurl: {
-        sel: 'img',
-        attr: 'src',
-        parse: src => `${baseUrl}/${src}`
+      currency: {
+        sel: '.order-amount',
+        // Warning : Not a standard space
+        parse: amount => amount.split(' ')[1]
       },
-      filename: {
-        sel: 'h3 a',
-        attr: 'title',
-        parse: title => `${title}.jpg`
+      momentDate: {
+        sel: 'div[class="order-label"]',
+        parse: string =>
+          moment(string.match(/\d{1,2}\/\d{2}\/\d{4}/), 'DD-MM-YYYY')
+      },
+      fileurl: {
+        sel: 'a[class="btn-enabled"]',
+        attr: 'href'
       }
     },
-    'article'
+    'div[class="panel panel-default"]'
   )
-  return docs.map(doc => ({
-    ...doc,
-    // the saveBills function needs a date field
-    // even if it is a little artificial here (these are not real bills)
-    date: new Date(),
-    currency: '€',
-    vendor: 'template',
-    metadata: {
-      // it can be interesting that we add the date of import. This is not mandatory but may be
-      // usefull for debugging or data migration
-      importDate: new Date(),
-      // document version, usefull for migration after change of document structure
-      version: 1
-    }
-  }))
-}
-
-// convert a price string to a float
-function normalizePrice(price) {
-  return parseFloat(price.replace('£', '').trim())
+  return docs.map(doc => {
+    ;(doc.filename =
```

Comment:
You can use object spread to make this more readable : 
```
docs.map(doc => {...doc,  
       filename: `...`;
       date: ....
       vendor: ...
       momentDate: undefined
})

Reply:
Don't mutate an object you did not create. This is hard to maintain because of closure and other unwanted side-effects.